frontend/app/src/pages/main/managed-workers/$managed-worker/components/update-form.tsx

```diff
@@ -583,6 +633,308 @@ export default function UpdateWorkerForm({
                   {memoryMbError && (
                     <div className="text-sm text-red-500">{memoryMbError}</div>
                   )}
+                  <Label>Scaling Method</Label>
+                  <Tabs
+                    defaultValue="static"
+                    value={hasAutoscaling ? 'autoscaling' : 'static'}
```

Comment:
I think this and the one below look basically the same? could we factor it out into a component?

Reply:
You mean the `<Label /> + <Tabs />`?

Reply:
ah sorry no, I meant the whole chunk of lines here (everything down to 937)

Reply:
gotcha, yeah flagged this in a comment above, but I'm going to avoid refactoring these two components at the moment, there's a lot that we need to pull out into shared components. eventually I'd like to create a `CreateOrUpdateRuntimeConfigForm` that we can share.

Reply:
nice, sounds good to me 👍